### PR TITLE
Body vid + body occluder

### PR DESCRIPTION
**Pull request by gazuntype**
Screenshot 2022-08-23 at 18 03 29

### CODE REVIEW

File: Editor/Simulator/BodyTracking/BodyOccluder/nakedMan.fbx.meta [hunk 1]
```diff
@@ -0,0 +1,105 @@
+fileFormatVersion: 2
```
Comment (chornesays) [inline Editor/Simulator/BodyTracking/BodyOccluder/nakedMan.fbx.meta:1]: yeah.....can we rename nakedMan.fbx to literally anything else

File: Editor/Simulator/BodyTracking/BodyOccluder/TransparentOccluder.mat [hunk 2]
```diff
@@ -0,0 +1,92 @@
+%YAML 1.1
```
[the remark this answers is not in the document]
Comment (gazuntype) [reply Editor/Simulator/BodyTracking/BodyOccluder/TransparentOccluder.mat:1]: that one is in the template, not the plugin. artists can delete it etc etc.

File: Editor/Simulator/BodyTracking/BodyOccluder/nakedMan.fbx.meta (hunk 1 above)
Comment (gazuntype) [reply Editor/Simulator/BodyTracking/BodyOccluder/nakedMan.fbx.meta:1]: alrighty. 😂😂😂😂

File: Editor/Simulator/BodyTracking/BodyOccluder/TransparentOccluder.mat (hunk 2 above)
[the remark this answers is not in the document]
Comment (chornesays) [reply Editor/Simulator/BodyTracking/BodyOccluder/TransparentOccluder.mat:1]: mmmmmmmmmmmyes

**Pull request merged**